build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+build_dir=./build
+src_dir=./src
+
+test -d "$build_dir" || mkdir "$build_dir"
```

Comment:
nit `test -d "${build_dir} || mkdir -f "${build_dir}"`

Reply:
there is no `-f` option for `mkdir`